public/js/Board.js

```diff
@@ -17,23 +17,12 @@ class Board {
     }, styles);
 
     // Set up last render state to calculate diffs for optimization
-    this.panned = false;
     this.lastRenderStyles = {};
```

Comment:
Can you also delete this line? I don't think it's used anymore with the other deletions.